project/frontend/src/index.js

```diff
@@ -13,18 +13,27 @@ import MapContainer from './components/ResultsMap.js';
 import MatchResults from './pages/MatchResults.js';
 import PreferenceForm from './components/PreferenceForm.js';
 
+import RestaurantCard from './components/RestaurantCard.js';
+
 import * as serviceWorker from './serviceWorker';
 
+// TODO: remove
+import { restaurant, details } from './components/SampleRestaurant.js';
+
 ReactDOM.render(
   <React.StrictMode>
     <Authentication>
       <Router>
         <Header />
         <Switch>
-          <Route exact path='/' component={HomePage} />
+          <Route path='/card'>
+            <RestaurantCard restaurant={restaurant} details={details} />
+          </Route>
+
           <Route path='/find-match' component={PreferenceForm} />
           <Route path='/match-results' component={MatchResults} />
           <Route path='/mapRoute' component={MapContainer} />
+          <Route path='/' component={HomePage} />
```

Comment:
Did you move this to the bottom and get rid of "exact" so that any other url directs here? If so, I think leaving the exact path above and then using redirect at the bottom might be better but not sure https://reactrouter.com/web/api/Redirect

Reply:
I did. Yea I like your idea much better, I will change it.

Reply:
So using the Redirect causes some problems with the Authentication component for some reason, so I'm going to postpone this change for another PR, because I anticipate having to make some changes to Authentication.

Reply:
I will leave a TODO for now.